include/tuvx/linalg/linalg.inl

```diff
@@ -0,0 +1,83 @@
+
+#include <vector>
```

Comment:
```suggestion
```
Since this file is included directly in the `linalg.h` header, this isn't necessary here. However, you're free to leave it